blackbird_python/blackbird/tests/test_listener.py

```diff
@@ -35,6 +35,7 @@
 name test_name
 version 1.0
 target fock (num_subsystems=1, cutoff_dim=7, shots=10)
+type TDM (copies=1000)
```

Comment:
Just double checking, but is there a test file that doesn't have the `type` keyword arg? Just to ensure everything still works without the keyword present!

Reply:
Oh, maybe I should've removed this from here (since it's not actually tested anywhere). Should be OK though with or without the `type TDM (copies=1000)` line. It's set as an optional line the same way as `target`.